ntt/ntt.cuh

```diff
@@ -272,17 +228,26 @@ public:
             const auto gen_powers =
                 NTTParameters::all()[gpu.id()]->partial_group_gen_powers;
 
-            bit_rev(aux_data, domain_data, lg_domain_size, gpu);
+            event_t sync_event;
+
+            if (aux_out != nullptr) {
+                bit_rev(aux_data, domain_data, lg_domain_size, gpu);
```

Comment:
`aux_size` would be better instead of `domain_size`. And it would be more consistent...

Reply:
What am I talking about? Brain fart! Sorry about the distraction :-(